docs/running-locally.md

```diff
@@ -101,6 +101,17 @@ If you made changes to the executor, you need to build the image:
 make argoexec-image
 ```
 
+Or you can use parameters to compile images that support specific platforms or multiple platforms with `TARGET_PLATFORM`:
+
```

Comment:
These are environment variables, not parameters
```suggestion
You can use the `TARGET_PLATFORM` environment variable to compile images for specific platforms:
```
- can also simplify this, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)